tests/kafkatest/tests/client/quota_test.py

```diff
@@ -128,23 +128,38 @@ def min_cluster_size(self):
     @cluster(num_nodes=5)
     @matrix(quota_type=[QuotaConfig.CLIENT_ID, QuotaConfig.USER, QuotaConfig.USER_CLIENT], override_quota=[True, False])
     @parametrize(quota_type=QuotaConfig.CLIENT_ID, consumer_num=2)
-    def test_quota(self, quota_type, override_quota=True, producer_num=1, consumer_num=1):
+    @parametrize(quota_type=QuotaConfig.CLIENT_ID, old_throttling_for_broker=True)
```

Comment:
Can you replace `old_throttling_for_broker` with `old_broker_throttling_behavior`? Same for `old_throttling_for_client`.